c10/core/DispatchKeySet.h

```diff
@@ -107,6 +107,9 @@ class DispatchKeySet final {
   uint64_t raw_repr() {
     return repr_;
   }
+  constexpr uint64_t raw_repr() const {
+    return repr_;
+  }
```

Comment:
There's reason to add a const overload here; the const variant is also valid in contexts where you have a mutable ref. Just delete the old accessor on line 107